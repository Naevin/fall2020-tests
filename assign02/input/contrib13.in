var a, b, c, d;
b = 1;
c = 0;
if (b || c) {
  a = c - b;
}
if ((c = b + a) || println(c)) {
  // no-op
} else {
  d = 1;
}

function squirt(a) {
  a + 1;
}

d - squirt(0) || println;
