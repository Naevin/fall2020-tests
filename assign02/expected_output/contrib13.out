0
Result: 1
